app/controllers/hosts_controller.rb

```diff
@@ -421,6 +422,15 @@ def multiple_destroy
   def multiple_build
   end
 
+  def rebuild_tftp
```

Comment:
It has similar purpose to multiple_build, it only shows a table of selected hosts and prompts user for confirmation.